app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb

```diff
@@ -1,13 +1,18 @@
 <% text_for :title do %>
-  <% if calculator.passport_country_in_electronic_visa_waiver_list? %>
+  <% if calculator.passport_country_in_electronic_visa_waiver_list? || calculator.passport_country_is_saudi_arabia_or_bahrain? %>
     You’ll need an electronic visa waiver (EVW) or a Visitor in Transit visa
   <% else %>
     You’ll need a visa to pass through the UK in transit
   <% end %>
 <% end %>
 
 <% govspeak_for :body do %>
-  <% if calculator.passport_country_in_electronic_visa_waiver_list? %>
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    You must either apply for:
+
+    an [EVW](/get-electronic-visa-waiver) - you can apply for this from 1 June 2022
```

Comment:
Change:

an [EVW] 

to:

+ an [electronic visa waiver] 

Reason:

To make the link text more accessible. Also bullet markdown is missing.